target/linux/ramips/image/mt7621.mk

```diff
@@ -342,6 +342,13 @@ define Device/dlink_dir-2660-a1
 endef
 TARGET_DEVICES += dlink_dir-2660-a1
 
+define Device/dlink_dir-853-a3
+  $(Device/dlink_dir-xx60-a1)
```

Comment:
We should find a better name for this common definition ...

Reply:
Maybe something like Dlink-nand? It seems all the dlink routers on the mt7621 with nand flash share the same partition layout. If possible maybe keep the dtsi the same and rename in the mt7621.mk